frontend/src/main/scala/bloop/testing/LoggingEventHandler.scala

```diff
@@ -132,7 +133,15 @@ class LoggingEventHandler(logger: Logger) extends BloopTestSuiteEventHandler {
       }
     }
 
-    logger.info("===============================================")
+    logger.info(delimiter)
+  }
+
+  def getTerminalWidth: Int = {
```

Comment:
We should put it as a value in an object, otherwise we'll be invoking this a lot.

I also wonder if it's really what we want, if we have a terminal for an entire screen, we would get a lot of unneccessary additional '='

Maybe instead we should make it match the longest part of the produced string? OR half a screen?

Reply:
Hi excuses for the late response.
I think it's pre-set to have length 80 regardless. The change is merely to reduce the manual repeat ====== in the test cases